microsoftgraph/microsoft-graph-explorer-v4
# Fix: Apperaing/Disappearing popup

## Overview

This PR 

- stops the OCPS background call
- disables feedback collection for AAD user's for compliance reasons

fixes #1328 

**Testing**

1. Log in with your AAD account
2. Observe if a pop up appears/disappears
3. Also observe the feedback button remains hidden
